<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QMAddMultipleTrainDataDialog</class>
 <widget class="QDialog" name="QMAddMultipleTrainDataDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>426</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mehrfacheinträge hinzufügen</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Mitarbeiter</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Gruppe</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="cbEmployeeGroup">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>25</height>
          </size>
         </property>
         <property name="editable">
          <bool>true</bool>
         </property>
         <property name="maxVisibleItems">
          <number>20</number>
         </property>
         <property name="insertPolicy">
          <enum>QComboBox::NoInsert</enum>
         </property>
         <property name="minimumContentsLength">
          <number>20</number>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QToolButton" name="tbAddEmployeeFromGroup">
         <property name="minimumSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="../rcs/icons.qrc">
           <normaloff>:/icons/line-awesome/svg/icons/line-awesome/plus-circle-solid.svg</normaloff>:/icons/line-awesome/svg/icons/line-awesome/plus-circle-solid.svg</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Einzel</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="cbSingleEmployee">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>25</height>
          </size>
         </property>
         <property name="editable">
          <bool>true</bool>
         </property>
         <property name="maxVisibleItems">
          <number>20</number>
         </property>
         <property name="insertPolicy">
          <enum>QComboBox::NoInsert</enum>
         </property>
         <property name="minimumContentsLength">
          <number>20</number>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QToolButton" name="tbAddEmployee">
         <property name="minimumSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="../rcs/icons.qrc">
           <normaloff>:/icons/line-awesome/svg/icons/line-awesome/plus-circle-solid.svg</normaloff>:/icons/line-awesome/svg/icons/line-awesome/plus-circle-solid.svg</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="2" column="0" rowspan="3" colspan="2">
        <widget class="QListWidget" name="lwEmployees">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QToolButton" name="tbRemoveEmployee">
         <property name="minimumSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="../rcs/icons.qrc">
           <normaloff>:/icons/line-awesome/svg/icons/line-awesome/minus-circle-solid.svg</normaloff>:/icons/line-awesome/svg/icons/line-awesome/minus-circle-solid.svg</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QToolButton" name="tbClear">
         <property name="minimumSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="../rcs/icons.qrc">
           <normaloff>:/icons/line-awesome/svg/icons/line-awesome/undo-solid.svg</normaloff>:/icons/line-awesome/svg/icons/line-awesome/undo-solid.svg</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>255</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="widget">
      <attribute name="title">
       <string>Vorgaben</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Schulung</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cbTrain">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>25</height>
            </size>
           </property>
           <property name="editable">
            <bool>true</bool>
           </property>
           <property name="maxVisibleItems">
            <number>20</number>
           </property>
           <property name="insertPolicy">
            <enum>QComboBox::NoInsert</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Datum</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCalendarWidget" name="cwTrainDate">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_8">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Status</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cbTrainState">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>25</height>
            </size>
           </property>
           <property name="editable">
            <bool>true</bool>
           </property>
           <property name="maxVisibleItems">
            <number>20</number>
           </property>
           <property name="insertPolicy">
            <enum>QComboBox::NoInsert</enum>
           </property>
           <property name="minimumContentsLength">
            <number>0</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbCancel">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>Abbrechen</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbCreate">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>Erstellen</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../rcs/icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pbCancel</sender>
   <signal>clicked()</signal>
   <receiver>QMAddMultipleTrainDataDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>262</x>
     <y>529</y>
    </hint>
    <hint type="destinationlabel">
     <x>212</x>
     <y>276</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbCreate</sender>
   <signal>clicked()</signal>
   <receiver>QMAddMultipleTrainDataDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>368</x>
     <y>529</y>
    </hint>
    <hint type="destinationlabel">
     <x>212</x>
     <y>276</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tbClear</sender>
   <signal>clicked()</signal>
   <receiver>QMAddMultipleTrainDataDialog</receiver>
   <slot>resetList()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>389</x>
     <y>196</y>
    </hint>
    <hint type="destinationlabel">
     <x>212</x>
     <y>276</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tbAddEmployee</sender>
   <signal>clicked()</signal>
   <receiver>QMAddMultipleTrainDataDialog</receiver>
   <slot>addEmployee()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>388</x>
     <y>107</y>
    </hint>
    <hint type="destinationlabel">
     <x>212</x>
     <y>276</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tbAddEmployeeFromGroup</sender>
   <signal>clicked()</signal>
   <receiver>QMAddMultipleTrainDataDialog</receiver>
   <slot>addEmployeeFromGroup()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>388</x>
     <y>63</y>
    </hint>
    <hint type="destinationlabel">
     <x>212</x>
     <y>276</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tbRemoveEmployee</sender>
   <signal>clicked()</signal>
   <receiver>QMAddMultipleTrainDataDialog</receiver>
   <slot>removeEmployee()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>389</x>
     <y>152</y>
    </hint>
    <hint type="destinationlabel">
     <x>212</x>
     <y>276</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>addEmployeeFromGroup()</slot>
  <slot>addEmployee()</slot>
  <slot>removeEmployee()</slot>
  <slot>resetList()</slot>
 </slots>
</ui>
